ci/environment_pip.txt

```diff
@@ -0,0 +1,7 @@
+myst-nb==0.12.3
+sphinx-book-theme
+sphinx-panels
+matplotlib
+pandas
+pyyaml
+pre-commit
```

Comment:
Do we actually need precommit for readthedocs?

Reply:
Good question, I just copied all of the dependencies from the `environment.yaml` but maybe if we're having 2 locations for dependencies anyway - they can be different?